### ISSUE TITLE
[Feature] : multiple launguage support , 

### ISSUE THREAD

**Issue by Jashwanth-rit** [Yashgabani845/hiring-portal#208]
Description: We need to add multiple language support to our application to enhance user experience and accessibility. Users should be able to switch between different languages seamlessly.

![Screenshot 2024-10-21 181704](https://github.com/user-attachments/assets/c97f27ef-da31-43cc-8023-9b1cda62c789)

@samyak-aditya Please Assign this to me by adding Gssoc and hacktoberfest Tag

**Comment by Yashgabani845** [comment]
@Jashwanth-rit it is done
